debian/patches/broadcom-firmware.patch

```diff
@@ -0,0 +1,152 @@
+Index: b/tools/hciattach.c
+===================================================================
+--- a/tools/hciattach.c	2016-05-26 11:51:11.000000000 -0500
++++ b/tools/hciattach.c	2020-03-08 21:04:47.725900647 -0500
+@@ -62,6 +62,7 @@
+ 	char *bdaddr;
+ 	int  (*init) (int fd, struct uart_t *u, struct termios *ti);
+ 	int  (*post) (int fd, struct uart_t *u, struct termios *ti);
++	char *fwdir;
+ };
+ 
+ #define FLOW_CTL	0x0001
+@@ -275,7 +276,7 @@
+ 
+ static int bcm43xx(int fd, struct uart_t *u, struct termios *ti)
+ {
+-	return bcm43xx_init(fd, u->init_speed, u->speed, ti, u->bdaddr);
++	return bcm43xx_init(fd, u->init_speed, u->speed, ti, u->bdaddr, u->fwdir);
```

Comment:
BlueZ uses the Linux coding style. Thus, please wrap this line (80 chars).

Reply:
This line is 79 characters long when the patch is applied

Reply:
Well, [tabs are 8 characters](https://www.kernel.org/doc/html/v4.10/process/coding-style.html#indentation) and thus the line is 82 characters long, not 78.